src/EnergyPlus/UnitarySystem.cc

```diff
@@ -14697,27 +14512,29 @@ namespace UnitarySystems {
             state.dataAirLoop->AirLoopAFNInfo(AirLoopNum).LoopOnOffFanPartLoadRatio = this->FanPartLoadRatio;
             state.dataAirLoop->AirLoopAFNInfo(AirLoopNum).LoopCompCycRatio = this->m_CycRatio;
         }
-
         if (this->m_FirstPass) {
-
-            if (!state.dataGlobal->SysSizingCalc) {
-
-                if (CurOASysNum > 0) {
-                    OASysEqSizing(CurOASysNum).AirFlow = false;
-                    OASysEqSizing(CurOASysNum).CoolingAirFlow = false;
-                    OASysEqSizing(CurOASysNum).HeatingAirFlow = false;
-                    OASysEqSizing(CurOASysNum).Capacity = false;
-                    OASysEqSizing(CurOASysNum).CoolingCapacity = false;
-                    OASysEqSizing(CurOASysNum).HeatingCapacity = false;
-                    this->m_FirstPass = false;
-                } else if (state.dataSize->CurSysNum > 0) {
-                    state.dataAirLoop->AirLoopControlInfo(state.dataSize->CurSysNum).UnitarySysSimulating = false;
-                    DataSizing::resetHVACSizingGlobals(state, state.dataSize->CurZoneEqNum, state.dataSize->CurSysNum, this->m_FirstPass);
-                } else if (state.dataSize->CurZoneEqNum > 0) {
-                    DataSizing::resetHVACSizingGlobals(state, state.dataSize->CurZoneEqNum, state.dataSize->CurSysNum, this->m_FirstPass);
-                } else {
-                    this->m_FirstPass = false;
+            if (AirLoopNum > -1) {
```

Comment:
OutdoorAirUnit passes AirLoopNum = -1. Do not reset globals set by OutdoorAirUnit (this objects parent).